distributor-api-v1/operations.md

```diff
@@ -442,18 +433,40 @@ If the hotel does not use any payment gateway, the value is null. If it does, th
 * `Once`
 * `Daily`
 
-#### CurrencyValues   <a id="currencyvalues"></a>
+#### Amount <a id="amount"></a>
 
-An object where field names correspond to currency ISO codes and field values to amounts. Only currencies that the hotel accepts are listed, for example:
+An object that represents a structure that holds gross price, net price and tax values.
```

Comment:
this information was lost after change field names correspond to currency ISO codes and field values to amounts . What do you think about putting it back? It seems it's useful info

Reply:
Changed it a bit. I think this is better.